docs/nut-names.txt

```diff
@@ -1059,6 +1059,10 @@ to last as part of NUT standard protocol in the long run.
     | Put UPS in High Efficiency (aka ECO) mode
 | experimental.ecomode.disable      | usbhid-ups => mge-hid (Eaton/MGE)
     | Take the UPS out of High Efficiency (aka ECO) mode
+| experimental.bypass.ecomode.start   | usbhid-ups => mge-hid (Eaton/MGE)
+    | Put UPS in Bypass mode then High Efficiency (aka ECO) mode
+| experimental.bypass.ecomode.stop   | usbhid-ups => mge-hid (Eaton/MGE)
+    | Take the UPS out of High Efficiency (aka ECO) mode after exiting Bypass mode
 | experimental.essmode.enable       | usbhid-ups => mge-hid (Eaton/MGE)
     | Put UPS in Energy Saver System (aka ESS) mode
 | experimental.essmode.disable      | usbhid-ups => mge-hid (Eaton/MGE)
```

Comment:
@masterwishx : while we are here (and discussed use of "start/stop" for these two operations earlier), now that I see the other ecomode enable/disable keywords in this context, I wonder if they should also be renamed to "start/stop" pattern.

I.e. do these instant commands actively change the state of the device (inverter mode) into ECO or ESS, or back into normal? Then start/stop is a better choice for them too. Enable/disable would be more about flipping a fail-safe switch to permit an active change by a later command...

If you agree that such a change would be correct - note also CMDDESC and the actual code :)